crates/biome_js_analyze/src/lint/nursery/no_qwik_use_visible_task.rs

```diff
@@ -0,0 +1,118 @@
+use biome_analyze::context::RuleContext;
+use biome_analyze::{Ast, Rule, RuleDiagnostic, RuleDomain, RuleSource, declare_lint_rule};
+use biome_console::markup;
+use biome_diagnostics::Severity;
+use biome_js_syntax::{AnyJsCallArgument, JsCallExpression, global_identifier};
+use biome_rowan::AstSeparatedList;
+use biome_rowan::TextRange;
+use biome_rule_options::no_qwik_use_visible_task::NoQwikUseVisibleTaskOptions;
+
+declare_lint_rule! {
+    /// Disallow useVisibleTask$() functions in Qwik components.
+    ///
+    /// This rule is intended for use in Qwik applications to prevent the use of
+    /// useVisibleTask$() functions which are not recommended in Qwik.
```

Comment:
```suggestion
    /// This rule is intended for use in Qwik applications to prevent the use of
    /// `useVisibleTask$()` functions, which are not recommended in Qwik.
```

The documentation doesn't explain why `useVisibleTask$` shouldn't be used. Can we update the docs so we can explain it?

Reply:
https://github.com/biomejs/biome/pull/6150/files#diff-49bd94d8efc02453c7f50dcbf876437a2c9c3fc29dd861b1f5bc37f5d5e18279R9

@ptkagori ^